controllers:
  robot_state_controller:
    type: robot_state_controller/RobotStateController
    publish_rate: 50
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  shooter_standard_controller:
    type: rm_shooter_controllers/ShooterStandardController
    joint_fiction_left_name: "joint_fiction_left"
    joint_fiction_right_name: "joint_fiction_right"
    joint_trigger_name: "joint_trigger"
    push_angle: 1.4
    block_effort: 5.8
    block_duration: 0.5
    block_speed: 0.9
    anti_block_angle: 0.5
    anti_block_error: 0.1
    qd_10: 420.0
    qd_15: 495.0
    qd_16: 498.0
    qd_18: 540.0
    qd_30: 772.0
    pid_fiction_l: { p: 0.002, i: 0.0, d: 0.0, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
    pid_fiction_r: { p: 0.002, i: 0.0, d: 0.0, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
    pid_trigger: { p: 20, i: 0.0, d: 1.0, i_max: 0.0, i_min: 0.0, antiwindup: true, publish_state: true }
    publish_rate: 50
